leaflet-treering.js

```diff
@@ -2386,6 +2390,14 @@ function ViewData(Lt) {
           if (length == 9.999) {
             length = 9.998;
           }
+
+          //Format length number into a string with trailing zeros
+          lengthAsAString = String(length);
+          while(lengthAsAString.length<5)
+          {
+            lengthAsAString+='0';
+          }
+
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd

Reply:
using that method should make things a little more readable

Reply:
I just changed it thank you for the tip!